src/app/src/components/FacilityDetailsItem.jsx

```diff
@@ -52,7 +53,7 @@ const FacilityDetailsItem = ({
                 <Typography className={classes.label}>{label}</Typography>
             </div>
             <FacilityDetailsDetail
-                primary={primary}
+                locationLabeled={locationLabeled}
```

Comment:
This change caused this bug:
<img width="1225" alt="Screenshot 2024-02-16 at 09 48 53" src="https://github.com/opensupplyhub/open-supply-hub/assets/58369841/a25669e6-8931-4e47-b8d5-97c5a15bfff9">

Reply:
@vladsha-dev All comments have been fixed, pls re-check.